src/app/technical-record/adr/tank-inpections/__snapshots__/tank-inpections.component.spec.ts.snap

```diff
@@ -74,7 +74,7 @@ exports[`TankInpectionsComponent should create view only with populated data 1`]
                 class="govuk-table__cell"
                 id="test-tc2IntermediateExpiryDate"
               >
-                 18/03/2020 
+                 2020-03-18 
```

Comment:
this is actually broken. it should be `18/03/2020` in the result

Reply:
I missed this one, apologies